src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/CSharpCodeFixVerifier`2+Test.cs

```diff
@@ -145,6 +145,20 @@ protected override ImmutableArray<CodeAction> FilterCodeActions(ImmutableArray<C
                 CodeActionsVerifier?.Invoke(actions);
                 return base.FilterCodeActions(actions);
             }
+
+            protected override ImmutableArray<(Project project, Diagnostic diagnostic)> SortDistinctDiagnostics(IEnumerable<(Project project, Diagnostic diagnostic)> diagnostics)
```

Comment:
We were hitting test failures without this change. This is taken from @sharwell's fix for the same issue in the roslyn-analyzers repo: https://github.com/dotnet/roslyn-analyzers/pull/6711/commits/08bc1be6069d8fcebeb4fdec7ec0d0296d1c400f#diff-1a7c1959a9933553162b52f66d5adad528386845c9bae46f6a85b8877ec1c23cR29